grails-app/controllers/au/org/emii/portal/WpsController.groovy

```diff
@@ -63,7 +64,7 @@ class WpsController extends HostVerifyingController {
                 job: [
                     uuid: params.uuid,
                     reportUrl: g.createLink(action: 'jobReport', absolute: true, params: params),
-                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)),
+                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)).withZone( DateTimeZone.forID( "Australia/Melbourne" )),
```

Comment:
Just to note, javascript dateTimes native-representation is utc/zulu time. So, if the server always sends UTC, the client only needs to use toLocaleString() or similar to apply the client's locale to extract the correct timezone formatted value.
```
d = new Date()
Date 2016-10-13T23:21:57.532Z
d.toUTCString()
"Thu, 13 Oct 2016 23:21:57 GMT"
d.toLocaleString()
"14/10/2016, 10:21:57 am"
```